mono/utils/mono-threads-state-machine.c

```diff
@@ -436,7 +429,6 @@ Returns one of the following values:
 - InitSelfResume: The thread is blocked on self suspend and should be resumed 
 - InitAsyncResume: The thread is blocked on async suspend and should be resumed
 - ResumeInitBlockingResume: The thread was suspended on the exit path of blocking state and should be resumed
-      FIXME: ResumeInitBlockingResume is just InitSelfResume by a different name.
```

Comment:
I think we still need this comment.

Reply:
I looked up the names and I didn’t see anything that would indicate they are the same.